cuisino/evaluation/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+#compiment and complain
+class Evaluation(models.Model):
```

Comment:
why dont we just have a role field for whether its complain or complement? thatd make things easier for us. 
also, why 3 text fields instead of just 1?
please review our models.py from users app and make this in that format.

Reply:
Also, we need a incrementer or something of the sort to count and affect the other models in other applications.

Reply:
Also, need the complaints to alert both parties and need to be approved by manager/superuser.

Reply:
I think i did that in ModelItem and Users Models